src/app/routes/topic/index.js

```diff
@@ -1,10 +1,17 @@
 import { TopicPage } from '#pages';
 import { topicPath } from '#app/routes/utils/regex';
 import { TOPIC_PAGE } from '#app/routes/utils/pageTypes';
+import isLive from '#app/lib/utilities/isLive';
 import getInitialData from './getInitialData';
 
+const path = [topicPath];
+
+if (!isLive()) {
+  path.push('/persian/afghanistan');
```

Comment:
Will leave it optional but based on @amoore108's feedback yday we could also do something like

`const path = [topicPath, !isLive() && '/persian/afghanistan'].filter(Boolean);`

as an alternative if you find this more readable.

Reply:
Oh aye, it's totally down to preference! Go for it